src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -77,6 +77,7 @@ class Task extends Component {
     const task = taskData;
     const assignedPursuanceId = task.assigned_to_pursuance_id;
     const { showChildren, showTaskForm } = this.state;
+    const leaf = !!!task.subtask_gids.length;
```

Comment:
@martytheemartian Triple? Interesting

Reply:
Probably would work with only one '!', I think ...
Amazon's new color scheme is ugly.